src/main/java/com/google/sps/servlets/MentorEvidenceServlet.java

```diff
@@ -72,4 +82,36 @@ private void updateMentorEvidence(String paragraph) {
       logger.log(Level.SEVERE, exception.getMessage(), exception);
     }
   }
+
+  /**
+   * Adds a list of approvers (currently only the admins) to the mentor in the database.
+   */
+  private void addApprovers() {
+    int userId = Utility.getUserId();
+    // Create array to store IDs of approvers.
+    int[] approvers = {1, 2, 3};
```

Comment:
why does it store 1, 2, and 3?

Reply:
I hardcoded us as the approvers. I added a TODO to later pull all admins (and I think we'll need an admin table).